### ISSUE TITLE
Official windows build ignores OSC

### ISSUE THREAD

**Issue by Ulibos** [falkTX/Carla#925]
On windows (Win 7 x64) sending OSC commands to Carla yields nothing. It looks like Carla won't even reserve the port (my python script binds to this port and listens to it without errors when Carla is running). Tested on official release 2.0 and on github's release 2.1 Beta 1 x32 (I can't launch x64 beta version, It fails with python34.dll conflict, but that's another story). Firewall is setup to ignore Carla.

**Comment by falkTX** [comment]
this might be obvious.. but did you enable osc support in the settings? (required for 2.1)
the conflict is due to old carla being still there in %tmp% dir, I will fix this for the next builds

**Comment by Ulibos** [comment]
Cleared up the temp. Carla x32 gives fails finding entry point for __gxx_personality_sj0 in libstdc++6.dll
Carla x64 now starts (I guess I made the same mistake earlier, firing up Carla 2.0 and then firing Carla 2.1 x64 in parallel, so it caused files mixup and python lib conflict). I enabled OSC, got the new static port and tried firing my script. It successfully binds with that port. Carla continues to ignore my test OSC packet but I can't guarantee that it's a valid one. The only reliable test for now is checking if port is occupied by Carla (and it's not).

**Comment by falkTX** [comment]
I suggest opening %tmp% and start CarlaDebug.exe from the 64bit carla directory.
With that, you will see the console output. so you will be able to see any errors that happen behind the scenes

**Comment by Ulibos** [comment]
Debug menu shows that OSC packets do pass. So Carla was receiving my test commands all along. Only issue is I formed a bad command. I was referring to the only plugin via /Carla/1 assuming that plugin indices start from 1 like in example. After fixing it to 0 it actually worked.
Thank you for your help. And for this amazing software.
